app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -103,7 +103,8 @@ def standardize_airtable_data(df: pd.DataFrame) -> pd.DataFrame:
 
     # Replace columns that should be floats with NaN from None
     # IMPORTANT: ind_sp and ind_se are percentages but stored as ints in airtable so must convert to decimal!
-    df[['ind_sp', 'ind_se']] = df[['ind_sp', 'ind_se']].replace({None: np.nan}) / 100
+    df[['ind_sp', 'ind_se']] = df[['ind_sp', 'ind_se']].replace({None: np.nan}).apply(pd.to_numeric, errors="coerce", axis=1).fillna(0).astype('int64') / 100
+    # df[['ind_sp', 'ind_se']] = df[['ind_sp', 'ind_se']].replace({None: np.nan}).astype('int64') / 100
```

Comment:
Remove the commented code.

Reply:
Whar does this change do?

Reply:
This change was for the csv generator script, an error was being thrown since we had numbers in those fields that weren't actually defined as a number datatype